src/__tests__/store.js

```diff
@@ -76,11 +78,11 @@ describe('Store', () => {
     );
   });
 
-  test('should use "em" as default highlighting tag', () => {
+  test('should default highlighting tags should be undefined', () => {
     const store = createStore();
 
-    expect(store.highlightPreTag).toEqual('<em>');
-    expect(store.highlightPostTag).toEqual('</em>');
+    expect(store.highlightPreTag).toEqual(undefined);
+    expect(store.highlightPostTag).toEqual(undefined);
```

Comment:
Because we get the state from the helper, and we push special HL tags for XSS.
This test ensures we properly return user provided tags if any, or undefined otherwise.